src/views/Home/components/Schedule/index.view.tsx

```diff
@@ -40,12 +64,12 @@ const ScheduleComponent: React.FC = () => {
           </div>
           <div className="Schedule__greyRect"></div>
           <div className="Schedule__eventContainer">
-            {scheduleItems[selectedDay].events.map(({ event, time }) => {
+            {scheduleItems[selectedDay].events.map(({ event, time, end }) => {
               return (
                 <div className="Schedule__event" key={event}>
                   <div className="Schedule__eventText">{event}</div>
                   <div className="Schedule__timeText">{time}</div>
-                  <div className="Schedule__greyRect--variant"></div>
+                  {renderDivider(end)}
```

Comment:
Interesting way to render a rule line; CSS could also make this easier with the `:last-child` & `:nth-child` selectors, but that would involve an event subcontainer for each time slot. In this case, [use the map index to conditionally render a rule line if we're at the last index of the event map](https://stackoverflow.com/questions/38364400/index-inside-map-function)

It's also a bit visually unclear if there's two workshops in a single time slot. I'd recommend leveraging the `end` variable you have to set the renderDivider function to provide a light-grey, dashed/dotted divider if there's another event in the same time slot